ingest/rules/prepare_ndjson.smk

```diff
@@ -41,6 +41,14 @@ def aggregate_gisaid_ndjsons(wildcards):
     """
     if len(config.get("gisaid_pairs", [])):
         GISAID_PAIRS = config["gisaid_pairs"]
+    elif config.get('s3_dst') and hasattr(checkpoints, "fetch_unprocessed_files"):
```

Comment:
Should change to `s3_src` since we want the source files:

```suggestion
    elif config.get('s3_src') and hasattr(checkpoints, "fetch_unprocessed_files"):
```

Reply:
Updated in [force-push](https://github.com/nextstrain/seasonal-flu/compare/24e9191398fdb490b25adcb1cd190ca9ae4906fc..c8c36207513ba73bf2cbf9257090cea1f8de8498)